ProcessMaker/Events/PermissionUpdated.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use ProcessMaker\Contracts\SecurityLogEventInterface;
+use ProcessMaker\Models\User;
+use ProcessMaker\Traits\FormatSecurityLogChanges;
+
+class PermissionUpdated implements SecurityLogEventInterface
+{
+    use Dispatchable, FormatSecurityLogChanges;
+
+    private $changedPermissions;
```

Comment:
Please define the corresponding type for each variable

Reply:
Done